LearnositySDK/Request/Init.cs

```diff
@@ -518,5 +527,111 @@ public void validate(string service, ref JsonObject securityPacket, string secre
                 throw new Exception("The `secret` argument must be a valid string");
             }
         }
+
+        /// <summary>
+        /// Adds a meta field with SDK information in it to the requestPacket
+        /// </summary>
+        public void addTelemetryData()
+        {
+            if (this.isTelemetryEnabled())
+            {
+                if (this.requestPacket.getJsonObject("meta") != null)
+                {
+                    JsonObject meta = this.requestPacket.getJsonObject("meta");
+                    meta.set("sdk", this.getSdkMeta());
+                    this.requestPacket.set("meta", meta);
+                }
+                else
+                {
+                    JsonObject meta = new JsonObject();
+                    meta.set("sdk", this.getSdkMeta());
+                    this.requestPacket.set("meta", meta);
+                }
```

Comment:
Should we move these two lines outside the if and remove lines 541, 542 to avoid code duplicity?

Reply:
Thank you for catching this Gonzalo. I have pushed an update to do what you suggested. :)